Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -139,6 +143,15 @@ class XMLHttpRequestBase {
     }
   }
 
+  _stringToArrayBuffer(str: string): ArrayBuffer {
+    var buffer = new ArrayBuffer(str.length);
+    var bufferView = new Uint8Array(buffer);
```

Comment:
@nicklockwood @satya164  So you guys don't want to support this feature for post iOS7 devices (94% iOS devices in market)?